devicetypes/zooz/zooz-zse44-temperature-humidity-xs-sensor.src/zooz-zse44-temperature-humidity-xs-sensor.groovy

```diff
@@ -0,0 +1,421 @@
+	0x70: 2,	// Configuration v4
+	0x71: 3,	// Notification v4
+	0x72: 2,	// ManufacturerSpecific
+	0x73: 1,	// Powerlevel
+	0x7A: 2,	// FirmwareUpdateMd v5
+	0x80: 1,	// Battery
+	0x84: 2,	// WakeUp
+	0x85: 2,	// Association v3
+	0x86: 1,	// Version v2
+	0x87: 1,	// Indicator v3
+	0x8E: 2,	// Multi Channel Association v4
+	0x9F: 1 	// Security 2
+]
+
+@Field static Map configParams = [
+	lowBatteryReports: [num:2, title:"Low Battery Reports", size:1, defaultVal:10, options:[10:"10% [DEFAULT]", 20:"20%", 30:"30%", 40:"40%", 50:"50%"]],
+	tempReportingThreshold: [num:3, title:"Temperature Reporting Threshold", size:1, defaultVal:10, range:"10..100", desc:"10..100 (10 = 1°)"],
+	tempReportingInterval: [num:16, title:"Temperature Reporting Interval", size:2, defaultVal:240, range:"0..480", desc:"0(disabled), 1..480(minutes)"],
+	tempUnit: [num:13, title:"Temperature Unit", size:1, defaultVal:1, options:[0:"Celsius", 1:"Fahrenheit [DEFAULT]"]],
+	tempOffset: [num:14, title:"Temperature Offset", size:1, defaultVal:100, range:"0..200", desc:"0..200 (0: -10°, 100: 0°, 200: +10°)"],
+	highTempThreshold: [num:5, title:"Heat Alert Temperature", size:1, defaultVal:120, range:"50..120", desc:"50..120(°)"],
+	lowTempThreshold: [num:7, title:"Freeze Alert Temperature", size:1, defaultVal:10, range:"10..100", desc:"10..100(°)"],
+	humidityReportingThreshold: [num:4, title:"Humidity Reporting Threshold", size:1, defaultVal:5, range:"1..50", desc:"1..50(%)"],
+	humidityReportingInterval: [num:17, title:"Humidity Reporting Interval", size:2, defaultVal:240, range:"0..480", desc:"0(disabled), 1..480(minutes)"],
+	humidityOffset: [num:15, title:"Humidity Offset", size:1, defaultVal:100, range:"0..200", desc:"0..200 (0: -10%, 100: 0%, 200: +10%)"],
+	highHumidityThreshold: [num:9, title:"High Humidity Alert Level", size:1, defaultVal:0, range:"0..100", desc:"0(disabled), 1..100(%)"],
+	lowHumidityThreshold: [num:11, title:"Low Humidity Alert Level", size:1, defaultVal:0, range:"0..100", desc:"0(disabled), 1..100(%)"]
+]
+
+@Field static Map temperatureSensor = [sensorType:1, scale:1]
+@Field static Map humiditySensor = [sensorType: 5, scale:0]
+@Field static Map temperatureAlarm = [name:"temperatureAlarm", notificationType:4, normalEvent:0, highEvent:2, lowEvent:6]
+@Field static Map humidityAlarm = [name:"humidityAlarm", notificationType:16, normalEvent:0, highEvent:2, lowEvent:6]
+@Field static int wakeUpInterval = 43200
+
+metadata {
+	definition (
+		name: "Zooz ZSE44 Temperature | Humidity XS Sensor",
+		namespace: "Zooz",
+		author: "Kevin LaFramboise (@krlaframboise)",
+		ocfDeviceType:"oic.d.thermostat",
+		vid: "064b47e2-afde-3228-89a4-480f0f55b3fd",
+		mnmn: "SmartThingsCommunity"
+	) {
+		capability "Sensor"
+		capability "Temperature Measurement"
+		capability "Relative Humidity Measurement"
+		capability "Battery"
+		capability "Refresh"
+		capability "Health Check"
+		capability "Configuration"
+		capability "platemusic11009.temperatureHumiditySensor"
```

Comment:
is this just so you can display two values in one tile?

Reply:
Yes, which is an important feature for this device.  I personally use one of these to monitor the humidity in my basement so I want to see that on the dashboard, but other users might only care about temperature so they'll want to see that on their dashboard.

As far as I know there's no way to display both values on the dashboard using built-in capabilities and there's also no way to dynamically change which one is show using a preference so using a custom capability to display them both is really the only option.